client/package.json

```diff
@@ -8,6 +8,7 @@
   },
   "dependencies": {
     "axios": "^1.7.2",
+    "emailjs-com": "^3.2.0",
```

Comment:
<img width="1254" alt="Screenshot 2024-07-15 at 10 04 15" src="https://github.com/user-attachments/assets/f98c2a45-8f36-4f5e-8a7d-04169d8c2550">

You can use alternative like SendGrid, Brevo, Nodemailer, MailChimp...